invenio_damap/services/config.py

```diff
@@ -1,14 +1,14 @@
 # -*- coding: utf-8 -*-
 #
 # Copyright (C) 2022 Graz University of Technology.
-# Copyright (C) 2024 TU Wien.
+# Copyright (C) 2023 TU Wien.
```

Comment:
did we go back in time?

Reply:
kinda yes, the v12 compat commits were addressed back in 2023 :stuck_out_tongue: